micro-ui/web/micro-ui-internals/packages/modules/pgr/src/pages/employee/new-inbox.js

```
@@ -0,0 +1,67 @@
+    setPageConfig(_.cloneDeep(configs))
+  }, [location])
+
+
+
+  if (!pageConfig || isLoading) return <Loader />
+
+  return (
+    <React.Fragment>
+      <Header styles={{ fontSize: "32px" }}>{t(updatedConfig?.label)}{location?.state?.count ? <span className="inbox-count">{location?.state?.count}</span> : null}</Header>
+      <div className="inbox-search-wrapper">
+        <InboxSearchComposer configs={updatedConfig}></InboxSearchComposer>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)